uci_ssh.go

```diff
@@ -0,0 +1,121 @@
+package uci
+
+import (
+	"bytes"
+	"log"
+	"os"
+	"path/filepath"
+	"sync"
+	"time"
+
+	"golang.org/x/crypto/ssh"
+)
+
+type SshTree struct {
+	host   string
+	config ssh.ClientConfig
+	client *ssh.Client
+	tree
+	sync.Mutex
+}
+
+func NewSshTree(username string, password string, host string) (t *SshTree) {
+	config := &ssh.ClientConfig{
+		User: username,
+		Auth: []ssh.AuthMethod{
+			ssh.Password(password),
+		},
+		// Non-production only
+		HostKeyCallback: ssh.InsecureIgnoreHostKey(),
+		Timeout:         5 * time.Second,
+	}
+
+	t = &SshTree{
+		config: *config,
+		host:   host,
+	}
+	var err error
+	t.client, err = ssh.Dial("tcp", t.host+":"+"22", &t.config)
```

Comment:
This doesn't allow non-standard ports and also breaks if `host` is an IPv6 address. I'd prefer for `host` to be fully qualified, similar to the `address` parameter for [`net.Dial`](https://pkg.go.dev/net#Dial).

```suggestion
	t.client, err = ssh.Dial("tcp", t.host, t.config)
```